src/ui/components/Icon/styles.scss

```diff
@@ -37,3 +38,26 @@
   background: url('./user-dark.svg') center no-repeat;
   background-size: contain;
 }
+
+.Icon-browse {
+  background: url('./bullet-list.svg') center no-repeat;
+  background-size: cover;
+  height: 20px;
+  width: 20px;
+}
+
+.Icon-themes {
+  background: url('./themes.svg') center no-repeat;
+  background-size: cover;
+  height: 50px;
+  min-width: 41.5px;
```

Comment:
@tofumatt I had adjusted these values to preserve the aspect ratio of the original icons. Also, I fixed one parameter to 50px (49.5px I guess for the extensions icon) and adjusted the other parameter accordingly because 50px seemed about right after being rendered on the screen.